### PR TITLE
Create workplace-program.md

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: operations/workplace/people/working-at-mattermost/workplace-program.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+{Insert link to sign up}
+
+This activity is managed by {person}.
+
+## Celebration and Sympathy Gifts
+
+At Mattermost, we like to share celebrations of life milestones and sympathy with our team members!
+
+If you know someone who is either
+Celebrating a special milestone such as wedding engagements, weddings and childbirth; or
```
Comment (justinegeffen) [inline operations/workplace/people/working-at-mattermost/workplace-program.md]:
```suggestion
- Celebrating a special milestone such as wedding engagements, weddings, and childbirth; or
```

File: operations/workplace/people/working-at-mattermost/workplace-program.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Undergoing a major life event such as the passing of a family member/significant other, or is injured, ill or hospitalized
```
Comment (justinegeffen) [inline operations/workplace/people/working-at-mattermost/workplace-program.md]:
```suggestion
- Undergoing a major life event such as the passing of a family member/significant other, or is injured, ill or hospitalized
```

File: operations/workplace/people/working-at-mattermost/workplace-program.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Remember them with a gift to show you have them in your mind. You can request to send a celebration or sympathy gift to a team member by completing the [Celebration and Sympathy Gifts Request form](https://docs.google.com/forms/d/1_ltFCvZhItVNWS6JV-glOnk1aU-d9oOfUZrW3zEiWhg/edit).
+
+Notes:
+
+  - Multiple requests for the same event are combined into a single gift.
+  - All gifts will be sent to the address on file on their Bamboo profiles
```
Comment (justinegeffen) [inline operations/workplace/people/working-at-mattermost/workplace-program.md]:
```suggestion
  - All gifts will be sent to the address on file on their Bamboo profile.
```

File: operations/workplace/people/working-at-mattermost/workplace-program.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  - Before submitting a request, please confirm with the team member that their shipping address and phone number is currently up to date on their Bamboo profiles, and ask them to update it if it is not current.
```
Comment (justinegeffen) [inline operations/workplace/people/working-at-mattermost/workplace-program.md]:
```suggestion
  - Before submitting a request, please confirm with the team member that their shipping address and phone number is currently up to date on their Bamboo profile, and ask them to update it if it is not current.
```

**Review by justinegeffen — DISMISSED** [review]
This great, thanks @jasonblais. :)

**Review by justinegeffen — DISMISSED** [review]

**Comment by hannaparks** [comment]
Hey @jasonblais, looks great!

Would we be able to get the yoga budget approval before this page goes live?

**Comment by jasonblais** [comment]
> Would we be able to get the yoga budget approval before this page goes live?

Yes, or alternatively we can publish without the yoga session if it'll need some extra time.

**Comment by hannaparks** [comment]
> > Would we be able to get the yoga budget approval before this page goes live?
> 
> Yes, or alternatively we can publish without the yoga session if it'll need some extra time.

No worries, let's keep it in there!

**Review by hannaparks — DISMISSED** [review]

**Review by justinegeffen — DISMISSED** [review]

**Comment by natalie-hub** [comment]
I made one word change suggestion, but otherwise this looks great! Thank you @hannaparks and @jasonblais for putting together and documenting!

**Review by natalie-hub — APPROVED** [review]
I'm so sorry, I should have added my update to the word here. The changes look good!

**Review by natalie-hub — DISMISSED** [review]

File: operations/workplace/people/working-at-mattermost/workplace-program.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+  - {Insert dates and times here}
+
+{Insert link to sign up}
+
+This activity is managed by Lynn Conway.
+
+### Video Game Hour
+
+Into video games? Then join your fellow team to battle for the best at 11:30am Palo Alto time each Friday.
```
Comment (hannaparks) [inline operations/workplace/people/working-at-mattermost/workplace-program.md:27]: Hey @jasonblais, we need to change the time to 10:30am Palo Alto time.

**Pull request merged**